app/src/main/res/layout/fragment_first_onboarding.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="48dp"
+        android:layout_marginStart="32dp"
+        android:layout_marginEnd="32dp"
+        android:fontFamily="@font/montserrat_bold"
+        android:text="@string/text_onboarding_1"
+        android:textColor="@color/landing_font_black"
+        android:textSize="26sp"
+        />
+
+    <ImageView
+        android:id="@+id/imageView"
```

Comment:
try to be more specific when giving ids to views. for example, illustration_image_view or something